cloud/deploys/dev_azure/civiform_config.sh

```diff
@@ -12,7 +12,6 @@ export RESIDENT_AUTH_PROVIDER="loginRadius"
 export APPLICATION_NAME="dev"
```

Comment:
This file should be deleted and added to .gitignore

Reply:
oo it is, maybe i have typo